README.md

```diff
@@ -1,77 +1,154 @@
+| **--validate**  | <img src="docs/images/metro/Validate.png" alt="concordance_metro" width="600"/>  | This mode compares two VCF files together to compute a summary of the differences between them. <br> This step uses [**Glimpse2**](https://odelaneau.github.io/GLIMPSE/glimpse2/index.html) concordance process.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
 
-<!-- TODO nf-core: Add a brief overview of what the pipeline does and how it works -->
+## Pipeline output
+
+To see the results of an example test run with a full size dataset refer to the [results](https://nf-co.re/phaseimpute/results) tab on the nf-core website pipeline page.
+For more details on the output files and reports, please refer to the [output documentation](https://nf-co.re/phaseimpute/output).
 
 ## Credits
 
-nf-core/phaseimpute was originally written by @louislenezet.
+nf-core/phaseimpute was originally written by Louis Le Nézet & Anabella Trigila.
+
+We thank the following people for their extensive assistance in the development of this pipeline:
+
+- Saul Pierotti
+- Eugenia Fontecha
+- Matias Romero Victorica
+- Hemanoel Passarelli
 
 ## Contributions and Support
 
 If you would like to contribute to this pipeline, please see the [contributing guidelines](.github/CONTRIBUTING.md).
 
-For further information or help, don't hesitate to get in touch on [Slack](https://nfcore.slack.com/channels/phaseimpute) (you can join with [this invite](https://nf-co.re/join/slack)).
+For further information or help, don't hesitate to get in touch on the [Slack `#phaseimpute` channel](https://nfcore.slack.com/channels/phaseimpute) (you can join with [this invite](https://nf-co.re/join/slack)).
+
+## Citations
+
+<!-- TODO nf-core: Add citation for pipeline after first release. Uncomment lines below and update Zenodo doi and badge at the top of this file. -->
+
+If you use nf-core/phaseimpute for your analysis, please cite it using the following doi: [10.5281/zenodo.XXXXXX](https://doi.org/10.5281/zenodo.XXXXXX)
+
+An extensive list of references for the tools used by the pipeline can be found in the [`CITATIONS.md`](CITATIONS.md) file.
 
-## Citation
+You can cite the main imputation methods as follows:
```

Comment:
I'd leave citations of other than nf-core publications in 'CITATIONS.md' eventually I'd write: "An extensive list of references for the tools used by the pipeline, including QUILT, GLIMPSE, and STITCH, can be found in the [`CITATIONS.md`](CITATIONS.md) file.

Reply:
Done in #152